src/Controller/DefaultController.php

```diff
@@ -186,8 +188,27 @@ protected function isAutoFsAvailable()
      */
     protected function clearCache()
     {
-        $realCacheDir = $this->cacheDir;
-        $this->cacheClearer->clear($realCacheDir);
+        try {
+            $commandAndParams = [
+                'command' => 'cache:clear',
+                '--env' => 'prod'
+            ];
```

Comment:
Cache clear command has an option for not warming up the cache after deletion. If you use it the next request will force the creation of the cache at the expense of taking a little more time.

Maybe this way you can get rid of the horrible `sleep`

Reply:
This doesn't work either. Tried with --no-warmup .